x/concentrated-liquidity/internal/math/math.go

```diff
@@ -140,3 +144,9 @@ func AddLiquidity(liquidityA, liquidityB sdk.Dec) (finalLiquidity sdk.Dec) {
 	}
 	return liquidityA.Add(liquidityB)
 }
+
+// MulRoundUp multiplies a by b and rounds up to the nearest integer
+// at precision end.
+func MulRoundUp(a, b sdk.Dec) sdk.Dec {
+	return a.MulTruncate(b).Add(smallestDec)
```

Comment:
what's the signifiance of adding smallest dec?

Reply:
By first truncating and then adding the smallest value, we turn this operation into multiplication that rounds up